File: Python/String/find_and_replace_in_string.py
# Find and replace in a given string
def find_and_replace_in_string(provided_string, find_string, replace_string):
    return provided_string.replace(find_string, replace_string)


def main():
    # Find and replace in a given string
    print(find_and_replace_in_string("Hello World!", "World", "Universe"))


main()

File: TypeScript/String/find-and-replace-in-string.ts
function main(): void {
    // Find and replace in a string
    console.log(findAndReplaceInString("My Name is John", "John", "Adam"))
}

main()

// Find a substring in a string and replace it with another string.
function findAndReplaceInString(content: string, find: string, replace: string): string {
    return content.replace(find, replace)
}